kakao/src/main/kotlin/com/agoda/kakao/configurator/ConfiguratorBuilder.kt

```diff
@@ -0,0 +1,98 @@
+package com.agoda.kakao.configurator
+
+import android.support.test.espresso.DataInteraction
+import android.support.test.espresso.ViewInteraction
+import android.support.test.espresso.web.sugar.Web
+import com.agoda.kakao.delegates.DataInteractionDelegate
+import com.agoda.kakao.delegates.ViewInteractionDelegate
+import com.agoda.kakao.delegates.WebInteractionDelegate
+
+class ConfiguratorBuilder private constructor(
```

Comment:
`Configurator.Builder` is better way for designing builders. It better should be the inner static class of the `Configurator`.

Reply:
`Configurator.Builder(history).something().build()`

Reply:
done